src/core/server/rendering/views/fonts.tsx

```diff
@@ -557,36 +554,11 @@ export const Fonts: FunctionComponent<Props> = ({ url, theme }) => {
     )
     .join('\n');
 
-  /*
-   * The default fonts are added as CSS variables, overriding OUI's, and then
-   * the CSS variables are consumed.
-   */
-  const fontRules = `
-    :root {
-      --font-text: "${fontText.family}", -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial,
-                   sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
-
-      --font-code: "${fontCode.family}", Consolas, Menlo, Courier, monospace;
-
-      --oui-font-family: var(--font-text);
-      --oui-code-font-family: var(--font-code);
-    }
-
-    code, pre, kbd, samp {
-      font-family: var(--font-code);
-    }
-    html, input, textarea, select, button {
-      font-family: var(--font-text);
-    }
-
-  `;
-
```

Comment:
These were moved to startup.js.hbs to be injected via js. However, I am concerned about FOUT.

Reply:
It's not FOUT because that's render blocking JS (although, it turns out the bootstrap.js is also effectively render blocking, because the page content is set to `display:none` until it finishes)